package.json

```diff
@@ -31,38 +31,38 @@
   ],
   "bugs": "https://github.com/forcedotcom/lwc-dev-mobile-core/issues",
   "dependencies": {
-    "@oclif/config": "^1.18.2",
-    "@salesforce/command": "^4.2.1",
-    "@salesforce/core": "^2.31.1",
-    "ajv": "^8.8.2",
+    "@oclif/config": "^1.18.6",
```

Comment:
I had an open question about `@oclif/config` specifically: their [repo](https://github.com/oclif/config) lists this package as deprecated, with the suggestion to move to [`@oclif/core`](https://github.com/oclif/core). Should we take this opportunity to do so? The `@oclif/core` repo has a [migration guide](https://github.com/oclif/core/blob/main/MIGRATION.md) we could use.

This package is only receiving security updates at this point, and not sure how long that will be the case.

Reply:
hmm... sure that's a good idea... let me take a look at that.